CHANGELOG.md

```diff
@@ -1,5 +1,63 @@
+* fix the cache max bytes buffering check ([#7181](https://github.com/confluentinc/ksql/pull/7181)) ([f383800](https://github.com/confluentinc/ksql/commit/f3838002fc4c8c085a3f62bfd34073e232416de4))
+* get all available restore commands even on poll timeout ([#6985](https://github.com/confluentinc/ksql/pull/6985)) ([28a7ba9](https://github.com/confluentinc/ksql/commit/28a7ba9a3d05a821eea702dcfaff046463a5394c))
+* incorporate lambdas into CoercionUtil and change LambdaVariable from Literal to Expression ([#7178](https://github.com/confluentinc/ksql/pull/7178)) ([c882303](https://github.com/confluentinc/ksql/commit/c882303f13de5afb6b3e023b3f21ddf59aa199ef))
+* ksql.service.id should not be usable as a query parameter ([#7192](https://github.com/confluentinc/ksql/pull/7192)) ([cc5cd81](https://github.com/confluentinc/ksql/commit/cc5cd81cadc7fb8fb7cbf34c9e98a3a567df746e))
+* Make pull query metrics apply only to pull and not also push ([#6944](https://github.com/confluentinc/ksql/pull/6944)) ([1db18b3](https://github.com/confluentinc/ksql/commit/1db18b3e8b161ae7315db67917bb1f5d89da8c2a))
+* prevent IOB when printing topics with a key/value with an empty string ([#7162](https://github.com/confluentinc/ksql/pull/7162)) ([177d0db](https://github.com/confluentinc/ksql/commit/177d0dbff72adcded82f9da9f9637b23fc6ffa3c))
+* Pull Queries: Avoids KsqlConfig copy with overrides since this is very inefficient ([#7193](https://github.com/confluentinc/ksql/pull/7193)) ([b36a3ce](https://github.com/confluentinc/ksql/commit/b36a3ce7d02dbeec7740d32801d443c4d7bca1e2))
+* quick fix for appender ([#7078](https://github.com/confluentinc/ksql/pull/7078)) ([8bc16b3](https://github.com/confluentinc/ksql/commit/8bc16b3f8b6d6a785f65c73c510180db443ef4aa))
+* return null on null input for timestamp math functions ([#7159](https://github.com/confluentinc/ksql/pull/7159)) ([1675221](https://github.com/confluentinc/ksql/commit/1675221e2b9342070a6210ffe33bf6a6eac1ad14))
+* script used in kci-170 is now exporting an array instead of string ([#7134](https://github.com/confluentinc/ksql/pull/7134)) ([986cbe3](https://github.com/confluentinc/ksql/commit/986cbe38238f05af11eb841801eb715088328496))
+* Table Scan: Filters out StreamsMetadata for wrong subtopology ([#7086](https://github.com/confluentinc/ksql/pull/7086)) ([9614720](https://github.com/confluentinc/ksql/commit/9614720fa423c3bebe78c13a2e1f5bf035be7cc3))
+* incompatible schema causing KsqlEngineTest.shouldNotFailIfAvroSchemaEvolvable failure ([#7052](https://github.com/confluentinc/ksql/pull/7052)) ([8c9e928](https://github.com/confluentinc/ksql/commit/8c9e9280bb1f76f8d4ef88bae7c3e33426ff3d48))
+
+### Reverts
+
+* Revert "Bump Confluent to 7.0.0-0, Kafka to 7.0.0-0" ([60ebe90](https://github.com/confluentinc/ksql/commit/60ebe90d2aa8fecc66e80239aacf0a49a81d8a5d))

```

Comment:
Seems like this can probably be removed, is that right?

Reply:
Yeah, let's remove this.